k8s-cleanup.tf

```diff
@@ -96,13 +96,8 @@ resource "kubernetes_deployment" "beekeeper_cleanup" {
           }
 
           env {
-            name = "BEEKEEPER_CONFIG"
-            value_from {
-              config_map_key_ref {
-                name = kubernetes_config_map.beekeeper[count.index].metadata.name
-                key  = "${local.cleanup_full_name}.properties"
-              }
-            }
+            name  = "BEEKEEPER_CONFIG"
+            value = base64encode(data.template_file.beekeeper_cleanup_config.rendered)
```

Comment:
Is this being switched from a config map to a deployment env to keep it more consistent to the ECS config?

Reply:
Yep, more details below :)